pkg/kudoctl/cmd/upgrade.go

```diff
@@ -157,7 +158,7 @@ func upgrade(newOv *v1alpha1.OperatorVersion, kc *kudo.Client, options *options)
 	}
 
 	// Change instance to point to the new OV and optionally update parameters
-	err = kc.UpdateInstance(options.InstanceName, options.Namespace, newOv.Name, options.Parameters)
+	err = kc.UpdateInstance(options.InstanceName, options.Namespace, util.String(ov.Name), options.Parameters)
```

Comment:
Hu? Why did this change from `newOv` to `ov`?

Reply:
good catch :)